### PR TITLE
Chapter1/homework/3 

### PR DESCRIPTION
**Pull request by Shareni**
Finished homework. I wasn't sure if it was cleaner to have each function separate, but I've left them together because they do the same thing, just for different variable types.

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                int age = GetInt("Amazing, and how old are you?");
+                float weight = GetFloat("Cool, and what's your weight?");
+                float height = GetFloat("Groovie, and how tall are you (in meters)?");
+                float bmi = Bmi(weight, height);
+
+                Console.WriteLine(name + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+                Console.WriteLine("Based on this information, your BMI is " + bmi);
+            }
+
+            GatherAndPrint("first");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:50]: It's quite a vague name. ``ProcessPersonInfo`` is better?

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static string GetString(string message)
```
[the remark this answers is not in the document]
Comment (Shareni) [reply Src/BootCamp.Chapter/Lesson3.cs]: Good point

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        }
+        public static float Bmi(float weight, float height)
+        {
+            float bmi = (weight / height / height);
+            return bmi;
+        }
+       
+        public static void Demo()
+        {
+            static void GatherAndPrint(string userNo)
```
[the remark this answers is not in the document]
Comment (Shareni) [reply Src/BootCamp.Chapter/Lesson3.cs]: I'm not sure I've done what you've meant. Should I also move the other functions below demo?

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (Shareni) [reply Src/BootCamp.Chapter/Lesson3.cs]: Sure is

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static string GetString(string message)
+        {
+            Console.WriteLine(message);
+            string string1= Convert.ToString(Console.ReadLine());
```
[the remark this answers is not in the document]
Comment (Shareni) [reply Src/BootCamp.Chapter/Lesson3.cs]: autopilot lol

**Pull request closed without merging**